src/lib/RelayOrderLib.sol

```diff
@@ -45,17 +48,54 @@ library RelayOrderLib {
         }
     }
 
+    /// @notice get the permissions necessary for the permit call
+    function toPermit(RelayOrder memory order)
```

Comment:
nit: this isn't returning the "permit", but just the TokenPermissions -- either rename `toTokenPermissions` or return the full permit

            ISignatureTransfer.PermitBatchTransferFrom({
                permitted: permissions,
                nonce: order.info.nonce,
                deadline: order.info.deadline
            }),